modules/openy_features/openy_node/modules/openy_node_class/openy_node_class.install

```diff
@@ -1,16 +1,33 @@
 <?php
+
 /**
- * @file OpenY Node Class install file.
+ * @file
+ * OpenY Node Class install file.
  */
 
+/**
+ * Implements hook_uninstall().
+ */
+function openy_node_class_uninstall() {
+  \Drupal::service('openy.modules_manager')
+    ->removeEntityBundle('node', 'node_type', 'class');
+  $configs = [
+    'pathauto.pattern.class',
```

Comment:
Is it possible to automatically detect configs that we need to remove in these uninstall functions?
The fear of mine is that with some updated contrib or core we'll get additional dependency, which will lead to unexpected situations on uninstall level

Reply:
For now this configs not contain any dependencies, but i think if we add module (where config stored) as dependency - this config should be deleted with module.

Reply:
Well, this is not a point I'm talking about. As I understand now we are gathering these config names manually for uninstalling hooks. Is there any way to automate?

Reply:
Different modules contain different number of related configs that should be deleted on uninstall manually and this configs sometimes also used different name conventions, so this hard to automate, anyway we need to provide list of this configs.

for example - https://github.com/ymcatwincities/openy/blob/decoupling/modules/openy_features/openy_node/modules/openy_node_blog/openy_node_blog.install#L16

Reply:
@podarok good point, I think it can work. Sounds like a separate task. We will try to investigate it separately and then just will do a small refactoring to replace these places with a list of configs. 

thank you for a great idea.